regions_www/m/server.R

```diff
@@ -333,9 +333,10 @@ shinyServer(function(input, output, session) {
    }
   }, priority = 3)
 
-  # Only requred to run if the region changes
+  # Only requred to run if the region changes (as that affects purpose) or the purpose changes (as that affects geographies)
   observe({
     region$current
+    input$purpose
```

Comment:
@AnnaGoodman1 I don't believe this should be needed

`region$purposes_present` and `region$geographies_present` only change when the `region$current` changes

https://github.com/npct/pct-shiny/blob/3374d9bb0b4a6b707431c19606a845ea48309aa6/regions_www/m/server.R#L284-L286

Would you mind explaining what this fixes?

Reply:
This (before it did not switch back to MSOAs):

![peek 2017-09-13 07-39](https://user-images.githubusercontent.com/1825120/30363070-ba5392b4-9856-11e7-941d-51fd78b4de67.gif)

Reply:
@nikolai-b : fixes this issue: https://github.com/npct/pct-shiny/issues/620 (sorry, should have cross-referenced)

Reply:
Right, that makes sense, thanks